### DISCUSSION TITLE
New configuration option

### DISCUSSION THREAD

**Discussion by jonny7737** [cline/cline#463] [Feature Requests]
The option to 'Always allow read-only operations' looks like it might be handy but many of my projects include a .env file with API keys. 

Always allowing read-only operations would be useful if the option to EXCLUDE certain files was possible.

Thanks for a great project.

**Comment by hannesrudolph** [comment]
Is your .env included in your .gitignore file?

**Comment by jonny7737** [comment]
Yes
